projects/elfutils/build.sh

```diff
@@ -72,6 +72,11 @@ if [[ "$SANITIZER" == undefined ]]; then
     sed -i 's/\(check_undefined_val\)=[0-9]/\1=1/' configure.ac
 fi
 
+if [[ "$SANITIZER" == memory ]]; then
+    CFLAGS+=" -U_FORTIFY_SOURCE"
```

Comment:
nit: just add a space before the +

Reply:
I agree it's ugly but bash would complain
```
 CFLAGS +=" -U_FORTIFY_SOURCE"
-bash: CFLAGS: command not found
```

Reply:
Oh lol nevermind then